### PR TITLE
changes made during integration test

### PR DESCRIPTION
**Pull request by RohanDudeja**

### CODE REVIEW

File: cmd/migrate/main.go [hunk 1]
```diff
@@ -16,6 +16,8 @@ var (
 )
 
 func main() {
+	//initialise config
+	config.BuildConfig()
```
Comment (mukesh0513) [inline cmd/migrate/main.go:20]: build config after args are validated. If invalid args are passed then do not need to read config.

File: internal/router/routes.go [hunk 2]
```diff
@@ -2,6 +2,7 @@ package router
 
 import (
 	"Tradeasy/internal/controller"
+	webSocket "Tradeasy/internal/controller"
```
Comment (mukesh0513) [inline internal/router/routes.go]: duplicate import.

File: cmd/migrate/main.go (hunk 1 above)
Comment (RohanDudeja) [reply cmd/migrate/main.go:20]: this is handled inside the config function.

File: internal/router/routes.go (hunk 2 above)
Comment (RohanDudeja) [reply internal/router/routes.go]: removed

**Review by mukesh0513 — APPROVED** [review]

**Pull request merged**